pattern-library/sass/patterns/_icons.scss

```diff
@@ -6,7 +6,6 @@
     color: currentColor;
     height: auto;
     width: auto;
-    font-family: $icon-font;
```

Comment:
It's fine to remove this and not specify any other font family?

@alisan617 I guess I'm a little hazy on what this fixes and how, can you link the ticket in the description of the PR?